ibm/mas_devops/roles/ocp_setup_mas_deps/tasks/main.yml

```diff
@@ -55,6 +55,29 @@
       - "OCP Install Version ................ {{ ocp_install_version }}"
       - "OCP Install Release ................ {{ ocp_install_release }}"
 
+#Getting cluster version details
```

Comment:
The comments state that we have `Removed desiredUpdate, updatechannel` from the ClusterVersion .. why are we doing this?  What does it achieve?  Is this something we want to do in every cluster, or is this something AWS specific to avoid undesirable behavious in AWS OCP installs?  What's the impact of making this change (add details in a comment in the code)

Reply:
This task was added for AWS OCP cluster. We have added that task to avoid automatic upgrades of OCP cluster. If anyone wants to do that they can manually upgrade the cluster to any desired version.